10-icmpv6-error/10-icmpv6-error.md

```diff
@@ -0,0 +1,330 @@
+## Goal: Check single-hop ICMP connectivity over IPv6
+
+*Note: This task requires to send some very specificly formated IPv6 packets. We
+recommend to use [scapy] for the construction of those packets in those tests. A
+packet can be sent e.g. using (you will require root permission for sending
+using link-layer frames with `sendp()`)*
+
+```py
+sendp(Ether(dst=DST_HWADDR) / IPv6(src=SRC_IPV6, dst=DST_IPV6) / UDP(dport=DST_PORT),
```

Comment:
This is much better. Stupid (or lazy guy) question: how to set a payload of 1452 as required [here](https://github.com/RIOT-OS/Release-Specs/pull/88/files#diff-cce1e7e335145eaef7f0b065d1b728bdR182) ?

Would the following work ?
```py
sendp(Ether(dst=DST_HWADDR) / IPv6(src=SRC_IPV6, dst=DST_IPV6) / UDP(dport=DST_PORT) / "x" * 1452 ,iface="tapbr0", timeout=0)
```

Reply:
Yepp

Reply:
Added something like that to the doc ;-).